### DISCUSSION TITLE
compile Boost v1.84 faild with gcc 4.8.5

### DISCUSSION THREAD

**Discussion by zdl474038463** [bfgroup/b2#365] [Q&A] [answered]
Hi there,
I compiled  Boost v1.75 success with gcc 4.8.5, but failed to  compile Boost v1.84. Can you help me?
[out.log](https://github.com/bfgroup/b2/files/14599484/out.log)

**Comment by grisumbras** [comment, marked as answer]
Since 1.84 the minimal C++ version required for many Boost libraries is C++11. GCC 4.8.5 doesn't enable it by default. Try building with `b2 cxxstd=0x`.

**Reply by zdl474038463** [reply]
building with b2 cxxflags="-std=c++11".
